internal/util/importutil/import_wrapper.go

```diff
@@ -254,20 +293,68 @@ func (p *ImportWrapper) Import(filePaths []string, rowBased bool, onlyValidate b
 		}
 	}
 
+	return p.reportPersisted()
```

Comment:
`debug.FreeOSMemory()` is only called once, when all data has been recovered/bulk-loaded.

Would it be a problem when there's a lot of data? Maybe we should call `debug.FreeOSMemory()` for every `MaxTotalSizeInMemory` size of data?

Reply:
debug.FreeOSMemory() returning all free memory back to the underlying system at once, this operation is expensive and can lead to latency spikes as it holds the heap lock through the whole process, especially in standalone mode.

Reply:
I have changed the logic to call debug.FreeOSMemory() after each segment imported.